### PR TITLE
Block www.wikia.com/fandom?p=* URLs

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php [hunk 1]
```diff
@@ -73,6 +73,7 @@ class WikiaRobots {
 	 */
 	private $blockedPaths = [
 		'/d/u/', // User pages for discussions
```
Comment (WikiaTech) [inline extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php:75]: INFO Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)

File: extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php [hunk 2]
```diff
@@ -73,6 +73,7 @@ class WikiaRobots {
...
+		'/fandom?p=', // Fandom old URLs
```
Comment (WikiaTech) [inline extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php:76]: INFO Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues
- MAJOR 4 major
- MINOR 2 minor
- INFO 4 info

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiaRobots.class.php#L147](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L147): Remove this unused "$wgRobotsTxtRemoveDeprecatedDirectives" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
2. MAJOR [WikiaRobots.class.php#L203](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L203): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)
3. MAJOR [WikiaRobots.class.php#L243](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L243): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
4. MAJOR [WikiaRobots.class.php#L245](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L245): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
5. MINOR [WikiaRobots.class.php#L52](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L52): Define a constant instead of duplicating this literal "allow" 6 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
6. MINOR [WikiaRobots.class.php#L205](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L205): Define a constant instead of duplicating this literal "noindex,nofollow" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
7. INFO [WikiaRobots.class.php#L87](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L87): Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)
8. INFO [WikiaRobots.class.php#L135](https://github.com/Wikia/app/blob/bf324a9165ee96c772dd2067e36bf1bbe54059f9/extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php#L135): Complete the task associated to this "TODO" comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)

**Comment by saipetch** [comment]
LGTM

**Pull request merged**